<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html>
 <head>
  <meta http-equiv="content-type" content="text/html; charset=UTF-8">
  <title>Get the cause of the exception</title>
<link media="all" rel="stylesheet" type="text/css" href="styles/03e73060321a0a848018724a6c83de7f-theme-base.css" />
<link media="all" rel="stylesheet" type="text/css" href="styles/03e73060321a0a848018724a6c83de7f-theme-medium.css" />

 </head>
 <body class="docs"><div class="navbar navbar-fixed-top">
  <div class="navbar-inner clearfix">
    <ul class="nav" style="width: 100%">
      <li style="float: left;"><a href="sdo-dataobject.gettypenamespaceuri.html">« SDO_DataObject::getTypeNamespaceURI</a></li>
      <li style="float: right;"><a href="sdo-list.insert.html">SDO_List::insert »</a></li>
    </ul>
  </div>
</div>
<div id="breadcrumbs" class="clearfix">
  <ul class="breadcrumbs-container">
    <li><a href="index.html">PHP Manual</a></li>
    <li><a href="ref.sdo.html">SDO 関数</a></li>
    <li>Get the cause of the exception</li>
  </ul>
</div>
<div id="layout">
  <div id="layout-content"><div id="sdo-exception.getcause" class="refentry">
 <div class="refnamediv">
  <h1 class="refname">SDO_Exception::getCause</h1>
  <p class="verinfo">(^)</p><p class="refpurpose"><span class="refname">SDO_Exception::getCause</span> &mdash; <span class="dc-title">
   Get the cause of the exception
  </span></p>

 </div>
 <div class="refsect1 description" id="refsect1-sdo-exception.getcause-description">
  <h3 class="title">説明</h3>
  <div class="methodsynopsis dc-description">
   <span class="methodname"><strong>SDO_Exception::getCause</strong></span>
    ( <span class="methodparam">void</span>
   ) : <span class="type"><a href="language.pseudo-types.html#language.types.mixed" class="type mixed">mixed</a></span></div>


  <div class="warning"><strong class="warning">警告</strong><p class="simpara">この関数は、
<em class="emphasis">実験的</em> なものです。この関数の動作・
名前・その他ドキュメントに書かれている事項は、予告なく、将来的な PHP
のリリースにおいて変更される可能性があります。
この関数は自己責任で使用してください。</p></div>

  <p class="para">
   Returns the cause of this exception or NULL if the cause is nonexistent or unknown.
   Typically the cause will be an SDO_CPPException object, which may
   be used to obtain additional diagnostic information.
  </p>

 </div>

 <div class="refsect1 parameters" id="refsect1-sdo-exception.getcause-parameters">
  <h3 class="title">パラメータ</h3>
  <p class="para">
   None.
  </p>
 </div>

 <div class="refsect1 returnvalues" id="refsect1-sdo-exception.getcause-returnvalues">
  <h3 class="title">返り値</h3>
  <p class="para">
   Returns the cause of this exception or NULL if the cause is nonexistent or unknown.
  </p>
 </div>


 


 


 


 


</div></div></div></body></html>